src/code42cli/cmds/legal_hold.py

```diff
@@ -140,8 +143,10 @@ def bulk(state):
 @sdk_options
 def add(state, csv_rows):
```

Comment:
These should also change to `bulk_add`/`bulk_remove` to match the others.

Reply:
In the other cases it was _needed_ because the modules had more than one function with the same name, but I agree consistency is good.